src/NrfTerminalCommander.ts

```diff
@@ -251,10 +251,8 @@ export default class NrfTerminalCommander implements ITerminalAddon {
      * moves the cursor to the beginning.
      */
     public clearUserInput(): void {
-        const charsToDelete = this.userInput.length - 1;
-        for (let i = 0; i <= charsToDelete; i += 1) {
-            this.backspace();
-        }
+        this.#terminal.write(ansi.cursorTo(this.#prompt.length - 2));
```

Comment:
Instead of calculating the length of the prompt, wouldn't it be easier to use `ansi.cursorTo(0)` to go back to the first column and output both, prompt and user input? 

That would probably require changing the prompt in such a way that it does not include the `\n\r`, but that seems better to me anyhow: Why should that be part of the prompt? It also made strange things like this `- 2` calculation necessary.

Oh, and a function that outputs both, prompt and current user input would also be something we could then use very well in the CM app instead of https://github.com/NordicPlayground/pc-nrfconnect-cellularmonitor/blob/22ec31b5ca499939dc24d97b45674d8fe78ba72c/src/Terminal/Terminal.tsx#L64-L68.

And if we replace `clearUserInput` with a `clearLine` that outputs `ansi.eraseLine + ansi.cursorTo(0)`, then we could also call that in https://github.com/NordicPlayground/pc-nrfconnect-cellularmonitor/blob/22ec31b5ca499939dc24d97b45674d8fe78ba72c/src/Terminal/Terminal.tsx#L72